Scripts/RemediationScripts/Remediate-StopExposingServiceFabricReverseProxyPort.ps1

```diff
@@ -0,0 +1,956 @@
+
+
+        if ($($ServiceFabricRemediated | Measure-Object).Count -gt 0)
+        {
+            Write-Host "Reverse proxy ports are not exposed publically now' for the following ServiceFabric(s) in the subscription:" -ForegroundColor $([Constants]::MessageType.Update)
+           
+            $ServiceFabricRemediated | Format-Table -Property $colsPropertyNew -Wrap
+
+            # Write this to a file.
+            $ServiceFabricRemediatedFile = "$($backupFolderPath)\RemediatedServiceFabric.csv"
+            $ServiceFabricRemediated | Export-CSV -Path $ServiceFabricRemediatedFile -NoTypeInformation
+
+            Write-Host "This information has been saved to" -NoNewline
+            Write-Host " [$($ServiceFabricRemediatedFile)]" -ForegroundColor $([Constants]::MessageType.Update) 
+            Write-Host "Use this file for any roll back that may be required." -ForegroundColor $([Constants]::MessageType.Info)
+        }
+
+        if ($($ServiceFabricSkipped | Measure-Object).Count -gt 0)
+        {
+            Write-Host "`nError while deleting the Load balancer rules exposing the Reverse proxy port for Service Fabric(s)in the subscription:" -ForegroundColor $([Constants]::MessageType.Error)
+            $ServiceFabricSkipped | Format-Table -Property $colsPropertyNew -Wrap
+            
+            # Write this to a file.
+            $ServiceFabricSkippedFile = "$($backupFolderPath)\SkippedServiceFabric.csv"
+            $ServiceFabricSkipped | Export-CSV -Path $ServiceFabricSkippedFile -NoTypeInformation
+            Write-Host "This information has been saved to"  -NoNewline
+            Write-Host " [$($ServiceFabricSkippedFile)]" -ForegroundColor $([Constants]::MessageType.Update)
+        }
+    }
+    else
+    {
+        Write-Host $([Constants]::DoubleDashLine)
+        Write-Host "[Step 4 of 4] Add LB Rule to expose Reverse proxy port Service Fabric(s) in the Subscription..."
+        Write-Host $([Constants]::SingleDashLine)
+        Write-Host "Skipped as -DryRun switch is provided." -ForegroundColor $([Constants]::MessageType.Warning)
+        Write-Host $([Constants]::DoubleDashLine)
+
+        Write-Host "`nNext steps:" -ForegroundColor $([Constants]::MessageType.Info)
+        Write-Host "*    Run the same command with -FilePath $($backupFile) and without -DryRun, Stop exposing Reverse proxy port for Service Fabric(s) listed in the file."
+    }
+}
+
+function Enable-ReverseProxyPortForServiceFabric
+{
+    <#
+        .SYNOPSIS
+        Roll back remediation done for 'Azure_ServiceFabric_DP_Dont_Expose_Reverse_Proxy_Port' Control.
+
+        .DESCRIPTION
+        Rolls back remediation done for 'Azure_ServiceFabric_DP_Dont_Expose_Reverse_Proxy_Port' Control.
+        Create Load balancer rule to expose Reverse proxy port on Service Fabric(s) in the Subscription. 
+        
+        .PARAMETER SubscriptionId
+        Specifies the ID of the Subscription that was previously remediated.
+        
+        .PARAMETER Force
```

Comment:
Remove force switch from here.

Reply:
Hi Tarun, as discussed, I have mentioned that this BRS does not support the force flag statement. Rest all params has been deleted.